.github/wiki/Contribution.md

```diff
@@ -0,0 +1,238 @@
+# Adding a new addon
+
+There are 3 types of addons.
+1. API
+2. Live
+3. Web
+
```

Comment:
We can also explain the addons structure by adding this image, what do you think @topnimble 
![Pasted_Image_08_03_2022__17_57](https://user-images.githubusercontent.com/11751745/157225788-9f8c28ad-058b-4994-851e-82c98854ddcd.png)

Reply:
Fixed in bf6fea9